Entity/SectionRepository.php

```diff
@@ -46,6 +88,7 @@ public function findByNavigationFromTree($navigationName, array $criteria = null
     /**
      * Find All From Tree
```

Comment:
You could simply remove this function as it is not used anymore.

findAllTree could be renamed to findAllForTree